run/cifs_archive/archive-clips.sh

```diff
@@ -37,6 +41,13 @@ function moveclips() {
         then
           log "Moved '$file_name'"
           NUM_FILES_MOVED=$((NUM_FILES_MOVED + 1))
+
+          # Every 10 files, send a wakeup command to the car via the Tesla API,
+          # to keep the Pi powered.
+          if (( $NUM_FILES_MOVED % 10 == 0 ))
+          then
+              keep_car_awake
```

Comment:
I think it would be better to make this `keep_ car_awake || true` or `if ! keep_car_awake; then log "failed to keep car awake"; fi` or similar, so that failure to access the tesla api (because it's down, or the token expired, or whatever) does not immediately abort the archive operation.

Reply:
Oh, would it do that? I'm a total amateur at shell programming, so I had no idea. I'll fix it.